crates/matrix-sdk-qrcode/README.md

```diff
@@ -6,27 +6,13 @@ interactive verification using [QR codes] in Matrix.
 This is probably not the crate you are looking for, it's used internally in the
 [matrix-sdk].
 
-If you still want to play with QR codes, here are a couple of helpful examples.
-
-### Decode an image
-
-```rust,no_run
-use image;
-use matrix_sdk_qrcode::{QrVerificationData, DecodingError};
-
-fn main() -> Result<(), DecodingError> {
-    let image = image::open("/path/to/my/image.png").unwrap();
-    let result = QrVerificationData::from_image(image)?;
-
-    Ok(())
-}
-```
+If you still want to play with QR codes, here is a helpful example.
 
 ### Encode into a QR code
 
 ```rust,no_run
 use matrix_sdk_qrcode::{QrVerificationData, DecodingError};
-use image::Luma;
+//use image::Luma;
```

Comment:
Why is this commented now?

Reply:
Otherwise tests are failing. The `README.md` is included in the Rust code. Despites the `no_run` marker, it is still “checked”, and since `image` is no longer a dependency, it fails.

Reply:
making `image` a `dev-dependency`?

Reply:
Indeed, make it a dev-dependency instead.

Reply:
Done.